templates/github/.github/workflows/scripts/build_python_client.sh.j2

```diff
@@ -14,9 +14,7 @@ pushd ../pulp-openapi-generator
 {%- for plugin in plugins %}
 rm -rf "{{ plugin.name | snake }}-client"
 
-# We need to copy that over to be visible in the container.
```

Comment:
This is no longer true. The gen-client script takes care of this now.
(And also stops to meddle with our version of the file...)